mir-module/src/main/resources/rules/rules-webpage.xml

```diff
@@ -70,6 +70,14 @@
                 <role>admin</role>
                 <role>editor</role>
             </or>
+            <or>
+                <not>
+                    <regex>webpage:/servlets/solr/select\?q=createdby:{CurrentUser}&amp;fq=objectType:mods.*</regex>
```

Comment:
Using plain { and } in a regexp seems wrong

Reply:
Thanks for the hint!